src/traces/scatter/attributes.js

```diff
@@ -6,8 +6,7 @@
 * LICENSE file in the root directory of this source tree.
 */
 
-
-var Plotly = require('../../plotly');
+var Drawing = require('../../components/drawing');
```

Comment:
This file is a dependency of the choropleth attributes file so it needed to be changed as well.